src/main/java/org/alfresco/heartbeat/InfoDataCollector.java

```diff
@@ -65,19 +64,19 @@
  *                  <li>minor: String - The minor version number, e.g. 1.<u>2</u>.3. {@link Descriptor#getVersionMinor()}</li>
  *                  <li>patch: String - The version revision number, e.g. 1.2.<u>3</u>. {@link Descriptor#getVersionRevision()}</li>
  *                  <li>hotfix: String - The version label. {@link Descriptor#getVersionLabel()}</li>
- *                  <li>build: String - The number of the build. {@link Descriptor#getVersionBuild()}</li>
+ *                  <li>build: String - The string which identifies the build. {@link Descriptor#getVersionBuild()}</li>
  *              </ul>
  *          </li>
  *          <li><b>schema:</b> Int - The schema number. {@link Descriptor#getSchema()}</li>
  *          <li><b>edition:</b> String - The edition. {@link Descriptor#getEdition()}</li>
  *          <li><b>deploymentMethod:</b> String - The deployment method used to deploy this Alfresco instance. {@link DeploymentMethodProvider#getDeploymentMethod()}</li>
- *          <li><b>os.vendor:</b> String - The name of the Operating System vendor. {@link System#getProperty(String)}</li>
- *          <li><b>os.version:</b> String - The version of the Operating System. {@link System#getProperty(String)}</li>
- *          <li><b>os.arch:</b> String - The architecture of the Operating System. {@link System#getProperty(String)}</li>
- *          <li><b>java.vendor:</b> String - The name of the Java vendor. {@link System#getProperty(String)}</li>
- *          <li><b>java.version:</b> String - The version of Java used. {@link System#getProperty(String)}</li>
- *          <li><b>user.language:</b> String - The language which this instance was installed with. {@link Locale#getLanguage()} </li>
- *          <li><b>user.timezone:</b> String - The GMT offset for the timezone of this instance, e.g. GMT+2. {@link DeploymentMethodProvider#getDeploymentMethod()}</li>
+ *          <li><b>osVendor:</b> String - The name of the Operating System vendor. {@link System#getProperty(String)}</li>
+ *          <li><b>osVersion:</b> String - The version of the Operating System. {@link System#getProperty(String)}</li>
+ *          <li><b>osArch:</b> String - The architecture of the Operating System. {@link System#getProperty(String)}</li>
+ *          <li><b>javaVendor:</b> String - The name of the Java vendor. {@link System#getProperty(String)}</li>
+ *          <li><b>javaVersion:</b> String - The version of Java used. {@link System#getProperty(String)}</li>
+ *          <li><b>userLanguage:</b> String - The language which this instance was installed with. {@link Locale#getLanguage()} </li>
+ *          <li><b>userTimezone:</b> String - The GMT offset for the timezone of this instance, e.g. GMT+2. {@link DeploymentMethodProvider#getDeploymentMethod()}</li>
```

Comment:
This is the `timezone id` with the latest changes ... but maybe you can still add the offset (please see my previous comment).